pkg/proxy/proxier.go

```diff
@@ -463,28 +464,29 @@ func (proxier *Proxier) OnUpdate(services []api.Service) {
 	glog.V(4).Infof("Received update notice: %+v", services)
 	activeServices := util.StringSet{}
 	for _, service := range services {
-		activeServices.Insert(service.Name)
-		info, exists := proxier.getServiceInfo(service.Name)
+		cachekey := types.NewNamespacedName(service.Namespace, service.Name).CacheKey()
```

Comment:
I don't understand the need for .CacheKey().  Why not just say .String() ?  CacheKey doesn't add any clarity and makes me want to go look it up because "it must be doing something clever"